tests/integration/helpers/cluster.py

```diff
@@ -1615,6 +1616,16 @@ def add_instance(
 
         if tag is None:
             tag = DOCKER_BASE_TAG
+        else:
+            if with_remote_database_disk:
+                raise Exception(
+                    f"Can't add instance '{name}': not support remote database disk with the old version {tag}"
+                )
+            with_remote_database_disk = False
+
+        if with_remote_database_disk is None:
+            with_remote_database_disk = True
```

Comment:
@tuanpach What do you think about using `False` as default and add one test that will explicitly test this new feature?